helm/rbac-operator/templates/rbac-tenant-admin.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tenant-admin
+  labels:
+    app: rbac-operator
+    giantswarm.io/service-type: "managed"
+rules:
+- apiGroups:
+  - ""
+  - application.giantswarm.io
+  - cluster.x-k8s.io
+  - infrastructure.giantswarm.io
+  resources:
+  - appcatalogs
+  - apps
+  - clusters
+  - machinedeployments
+  - awsclusters
+  - awsmachinedeployments
+  verbs:
```

Comment:
Maybe add patch as well?

Reply:
uff, mised that

Reply:
Can't we replace that with `"*"`?

Reply:
Yes, we can. Just wanted to be explicit here

Reply:
In this case this explicitly states you can do anything. But I won't argue.